app/bundles/PluginBundle/Views/FormTheme/Integration/_integration_details_featureSettings_leadFields_row.html.php

```diff
@@ -16,24 +16,49 @@
             <?php echo $view['translator']->trans($specialInstructions); ?>
         </div>
     <?php endif; ?>
-    <div class="form-group col-xs-12">
-        <?php echo $view['form']->errors($form); ?>
-        <?php $rowCount = 1; ?>
-        <?php foreach ($form->children as $child): ?>
-        <?php if ($rowCount++ % 2 == 1): ?>
+    <div class="contact-field form-group col-xs-12">
+        <?php $rowCount = 0; $indexCount = 1; $numberOfFields = ($form->offsetExists('update_mautic1')) ? 3 : 2; ?>
         <div class="row">
-        <?php endif; ?>
-            <div class="col-sm-6">
-                <?php echo $view['form']->row($child); ?>
-            </div>
-        <?php if ($rowCount++ % 2 == 1): ?>
+            <div class="mb-xs ml-lg pl-xs pr-xs col-sm-4 text-center"><h4><?php echo $view['translator']->trans('mautic.plugins.integration.fields'); ?></h4></div>
+            <?php if ($numberOfFields == 3): ?>
+            <div class="pl-xs pr-xs col-sm-2"></div>
+            <?php endif; ?>
+            <div class="pl-xs pr-xs col-sm-4 text-center"><h4><?php echo $view['translator']->trans('mautic.plugins.mautic.fields'); ?></h4></div>
+            <div class="pl-xs pr-xs col-sm-1 text-center"></div>
         </div>
-        <?php endif; ?>
-        <?php ++$rowCount; ?>
+        <?php echo $view['form']->errors($form); ?>
+        <?php foreach ($form->children as $child): ?>
+            <?php if ($rowCount % $numberOfFields == 0):  ?>
+                <div id="contact-<?php echo $rowCount; ?>" class="row<?php if ($rowCount > 0 && empty($child->vars['value'])) {
+    echo ' hide';
+} ?>">
+                <?php endif; ?>
+                    <?php ++$rowCount; ?>
+                    <div class="<?php if ($rowCount % $numberOfFields == 1) {
+    echo 'ml-lg';
+} ?> pl-xs pr-xs col-sm-<?php if ($rowCount % $numberOfFields == 2) {
+    echo '2 ml-xs';
+} else {
+    echo '4';
+} ?>">
+                        <?php echo $view['form']->row($child); ?>
```

Comment:
I think it's recommended to use the `<?php if (...) : ?>...<?php endif; ?>` syntax in the HTML templates.